parser/src/main/kotlin/tools/samt/parser/Nodes.kt

```diff
@@ -2,7 +2,11 @@ package tools.samt.parser
 
 import tools.samt.common.Location
 
-sealed class Node(val location: Location)
+abstract class Node(val location: Location) {
+    override fun toString(): String {
+        return ASTPrinter.dump(this)
```

Comment:
Would it be cleaner if the ASTPrinter would be part of the CLI? If we want to use ANSI-Colors etc. which are CLI-Specific it would make sense to not have a dependency on such things within the Parser-Nodes themselves.